.github/workflows/release.yml

```diff
@@ -1,101 +1,61 @@
-# If it detects a version bump on main, it will trigger a release.
-# If the workflow is started manually, it will skip the bump detection and attempt to publish.
-name: Release and publish
+name: Create GitHub release and publish to NPM
 
 on:
-  workflow_dispatch:
   push:
```

Comment:
Does it need this too?

<img width="282" alt="image" src="https://github.com/ERC725Alliance/erc725.js/assets/477945/0e31034b-59e3-4e94-a34c-f71cacd60ece">

Source: https://github.com/google-github-actions/release-please-action